packages/assets-controllers/src/Standards/NftStandards/ERC1155/ERC1155Standard.ts

```diff
@@ -161,13 +209,29 @@ export class ERC1155Standard {
     standard: string;
     tokenURI: string | undefined;
     image: string | undefined;
+    name: string | undefined;
+    symbol: string | undefined;
   }> => {
     const isERC1155 = await this.contractSupportsBase1155Interface(address);
 
     if (!isERC1155) {
       throw new Error("This isn't a valid ERC1155 contract");
     }
-    let tokenURI, image;
+
+    let tokenURI, image, symbol, name;
+
+    const [getNameResult, getSymbolResult] = await Promise.allSettled([
+      this.getAssetName(address),
+      this.getAssetSymbol(address),
+    ]);
+
+    if (getNameResult.status === 'fulfilled') {
+      name = getNameResult.value;
+    }
+
+    if (getSymbolResult.status === 'fulfilled') {
+      symbol = getSymbolResult.value;
+    }
```

Comment:
[nit] a little cleaner and the same I believe?
```suggestion
      name = getNameResult?.value;
      symbol = getSymbolResult?.value;
```

Reply:
value will only be defined if the promise fulfilled: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled